medusa/indexers/tvdbv2/tvdbv2_api.py

```diff
@@ -407,16 +391,16 @@ def _parse_images(self, sid):
 
         search_for_image_type = self.config['image_type']
 
-        logger.debug('Getting show banners for %s', sid)
+        log.debug('Getting show banners for {0}', sid)
         _images = {}
 
         # Let's get the different types of images available for this series
         try:
             series_images_count = self.config['session'].series_api.series_id_images_get(
                 sid, accept_language=self.config['language']
             )
-        except (ApiException, RequestException) as e:
-            logger.info('Could not get image count for showid: %s with reason: %r', sid, e.message)
+        except (ApiException, RequestException) as error:
+            log.info('Could not get image count for show id: {0} with reason: {1!r}', sid, error.message)
```

Comment:
repr isnt just for objects... for example a string would be displayed in quotes.